README.md

```diff
@@ -1,7 +1,19 @@
 
 # sbt-build-jobs
 
-This is a placeholder README.md for a new repository
+ [ ![Download](https://api.bintray.com/packages/hmrc/sbt-plugin-releases/sbt-build-jobs/images/download.svg) ](https://bintray.com/hmrc/sbt-plugin-releases/sbt-build-jobs/_latestVersion)
```

Comment:
These links 404. Are they expected to work in the future?

Reply:
Yeah, there is no build yet but once released to bintray they should work